nf_core/modules/lint/main_nf.py

```diff
@@ -203,16 +221,16 @@ def check_process_section(self, lines):
 
 def _parse_input(self, line):
     input = []
+    line = line.strip()
     # more than one input
     if "tuple" in line:
         line = line.replace("tuple", "")
         line = line.replace(" ", "")
-        line = line.split(",")
-
-        for elem in line:
-            elem = elem.split("(")[1]
-            elem = elem.replace(")", "").strip()
-            input.append(elem)
+        for idx, elem in enumerate(line.split(")")):
+            if elem:
+                elem = elem.split("(")[1]
+                elem = elem.split(",")[0].strip()
```

Comment:
I suspect regex here again would be nicer. Would be nice if you could pop in an example comment as you did above, as it's not clear to me from the code what is going on here really..

Reply:
Done in https://github.com/nf-core/tools/pull/1374/commits/0bfa000e0bd0f8f2502c429ca6b8c064b9b0d0cb